README.md

```diff
@@ -1,2 +1,9 @@
 # boost-ci
-Boost build for Windows
+
+Boost python build for Windows
+
+Master:
+[![Build status](https://ci.appveyor.com/api/projects/status/308fwmue023k8xac/branch/master?svg=true)](https://ci.appveyor.com/project/tiagocoutinho/boost-ci/branch/master)
```

Comment:
Would be nice to add some badges like this on PyTango, as part of tango-controls/pytango/pull/277

Reply:
Good idea!